lochness/utils/path_checker.py

```diff
@@ -89,7 +89,7 @@ def update_interviews_transcript_check(df: pd.DataFrame) -> pd.DataFrame:
     '''Check logics in rows for Interviews transcripts'''
     transcript_int_index = df[
             (df.modality=='Interviews') &
-            (df.subject.isin(['For_review', 'Approved']))].index
+            (df.subject.isin(['For_review', 'For review', 'Approved']))].index
```

Comment:
Did you knowingly duplicate `For review`?

Reply:
Thanks for catching this.

Reply:
[5035314](https://github.com/AMP-SCZ/lochness/pull/112/commits/50353147a1bc9bd1eb26d21ab16d0f2e3bb2c8d0)